apps/lead/models.py

```diff
@@ -260,6 +260,10 @@ def get_verified_entries_count(self):
             return self.verified_entries_count
         return self.entry_set.filter(verified=True).count()
 
+    def get_filtered_entries_count(self):
+        if hasattr(self, 'filtered_entries_count'):
+            return self.filtered_entries_count
+        return self.entry_set.count()
```

Comment:
This function is returning different values as entry_set doesn't have `filtered entries`.